src/billing/components/BillingPageContents.tsx

```diff
@@ -1,22 +1,23 @@
 // Libraries
 import React, {FC} from 'react'
+import {useSelector} from 'react-redux'
 
 // Components
 import BillingFree from 'src/billing/components/Free/Free'
 import BillingPayAsYouGo from 'src/billing/components/PayAsYouGo/PayAsYouGo'
 import MarketplaceBilling from 'src/billing/components/marketplace/MarketplaceBilling'
 
 // Utils
-import {useBilling} from 'src/billing/components/BillingPage'
+import {getQuartzMe} from 'src/me/selectors'
 
 const BillingPageContents: FC = () => {
-  const [{account}] = useBilling()
+  const quartzMe = useSelector(getQuartzMe)
 
-  if (!!account.marketplace) {
+  if (!!quartzMe.billingProvider) {
     return <MarketplaceBilling />
   }
```

Comment:
a payg account will always have a billing provider bc it represents who we send billing info to. This should probably be something more like below to check if its a marketplace.

```javascript
if (!!quartzMe.accountType == 'pay_as_you_go' && quartzMe.billingProvider !== 'zuora') {
  return <MarketplaceBilling />
}
```